go/vt/vttablet/tabletserver/schema/tracker.go

```diff
@@ -163,14 +170,72 @@ func (tr *Tracker) process(ctx context.Context) {
 	}
 }
 
+func (tr *Tracker) currentPosition(ctx context.Context) (mysql.Position, error) {
+	conn, err := tr.engine.cp.Connect(ctx)
+	if err != nil {
+		return mysql.Position{}, err
+	}
+	defer conn.Close()
+	return conn.MasterPosition()
+}
+
+func (tr *Tracker) isSchemaVersionTableEmpty(ctx context.Context) (bool, error) {
+	conn, err := tr.engine.GetConnection(ctx)
+	if err != nil {
+		return false, err
+	}
+	defer conn.Recycle()
+	result, err := withDDL.Exec(ctx, "select id from _vt.schema_version limit 1", conn.Exec)
+	if err != nil {
+		return false, err
+	}
+	if result == nil {
```

Comment:
This check is not needed. All APIs guarantee this will not be nil if there's no error.